emsdk.py

```diff
@@ -2710,14 +2710,29 @@ def extract_bool_arg(name):
 
   releases_info = load_releases_info()['releases']
 
+  def report_upstream_by_default():
+    print('''\
+** NOTICE **: The default SDK changed from `fastcomp` to `upstream`.
+If you have problems, or wish to revert back to fastcomp for some other reason
+you can add `-fastcomp` to explitly install that fastcomp-based
+SDK. .e.g ./emsdk install latest-fastcomp.
+''', file=sys.stderr)
+
   # Replace meta-packages with the real package names.
   if cmd in ('update', 'install', 'activate'):
     for i in range(2, len(sys.argv)):
       arg = sys.argv[i]
-      if arg in ('latest', 'sdk-latest', 'latest-64bit', 'sdk-latest-64bit', 'latest-fastcomp', 'latest-releases-fastcomp'):
+      if arg in ('latest', 'sdk-latest', 'latest-64bit', 'sdk-latest-64bit'):
+        # This is effectly the default SDK
+        report_upstream_by_default()
```

Comment:
I think we can report this only if the version is high enough? Can maybe share code with the existing check later down.

Reply:
The version check make sense below, but here we know that we want to unconditionally select `upstream` right?  We never want latest to point to fastcomp ever again, do we?  And if we do we would revert this code.